client/src/YourRequests.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import './App.css';
+import moment from 'moment';
+import * as api_fetch from './api'
+
+function YourRequests(props) {
+
+  useEffect( () => {
+      getData();
+    }, []);
+
+  // get requests from server
+  const getData = async () => {
+
+    api_fetch.loadYourRequests().then((data) => {
+
+      props.setUser(data.map(request => {
+        const a = request.date;
+        const b = [a.slice(0, 4), "-", a.slice(5, 7), "-", a.slice(8, 10)].join('');
+
+      return (
+        {
+          title: props.userName.name,
+          id: request.id,
+          start: moment(b).format("D.M.YYYY"),
+          status: request.status = request.status.toLowerCase(),
+          type: convertVacationType(request.type)
+        }
+      )
+    }))
+    }).catch(reason => {
+      alert(reason)
+    });
+}
+
+function convertVacationType(vacationType) {
```

Comment:
The very same function is already in `UpcomingRequests.js`. DRY (dont repeat yourself) - move the function to separate file, export it from there and import it wherever needed (i.e. in `UpcomingRequests.js` and here).